config/locales/en.yml

```diff
@@ -481,6 +481,7 @@ en:
     active_courses: Active Courses
     add_trainings: Please add student trainings to your assignment timeline. Assigning training modules is an essential part of Wiki Ed's best practices.
     alerts: Alerts
+    does_not_exist: The Item doesn't exist. Create a Sandbox page and assign yourself to that to receive suggestions.
```

Comment:
for courses with no ratings it throughs an error,  of missing.en.does_not_exist because the en.yml file does not contain a defintion for does_not_exist under courses